class/defaults.yml

```diff
@@ -10,106 +10,46 @@ parameters:
         repository: appuio/appuio-reporting
         tag: v0.17.0
 
-      erp_adapter:
-        registry: ghcr.io
-        repository: vshn/appuio-odoo-adapter
-        tag: v1.13.0
-
     schedules:
-      tenantmapping: '10 * * * *'
       backfill: '15 * * * *'
-      check_missing: '30 8-14 * * *'
-      invoice: '0 10 1 * *'
 
-    development_mode: false
     schedules_suspend:
-      tenantmapping: false
       backfill: false
-      check_missing: false
-      invoice: false
 
-    database:
-      url: postgres://$(DB_USER):$(DB_PASSWORD)@$(DB_HOST):$(DB_PORT)/$(DB_NAME)?$(DB_PARAMS)
-      parameters: ''
-    database_secret: {}
-    database_env: {}
+    development_mode: false
+    override_sales_order_id: null
 
     extra_volumes: {}
 
     prometheus:
       url: null
       org_id: null
 
-    erp_adapter:
-      url: null
-      invoice_title: APPUiO Cloud
-
-    backfill:
-      queries:
-        appuio_cloud_memory: true
-        appuio_cloud_loadbalancer: true
-        appuio_cloud_persistent_storage: true
-        appcat_services: true
+    odoo:
+      metered_billing_endpoint: null
+      oauth:
+        token_endpoint: null
+        client_id: null
+        client_secret: null
 
-    tenantmapping:
-      dry_run: true
-      metrics_selector: namespace="appuio-control-api-production"
+    rules: {}
```

Comment:
Do I understand correctly that we'll have to manage all the billing rules in the global defaults?

Reply:
Generally, I'm in favor of anything that reduces the amount of large YAML configurations that we need to manage somewhere

Reply:
well it'd be either in the component or in the hierarchy, but we'll have to manage it regardless...